### ISSUE TITLE
Errors when npm install and trying to run

### ISSUE THREAD

**Issue by KimHWBryan** [ajmeese7/spambot#5]
When I do "npm install", I get

npm WARN discord.js@11.3.2 requires a peer of bufferutil@^3.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of erlpack@discordapp/erlpack but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of node-opus@^0.2.7 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of opusscript@^0.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of sodium@^2.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of libsodium-wrappers@^0.7.3 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of uws@^9.14.0 but none is installed. You must install peer dependencies yourself.
npm WARN spambot@1.0.0 No repository field.

when I try to run (node bot,js)

Ready to level up!
(node:81152) UnhandledPromiseRejectionWarning: Error: Incorrect login details were provided.
    at WebSocketConnection.client.ws.connection.once.event (C:\Users\Bryan Hyunwoo Kim\Desktop\spambot-master\spambot\node_modules\discord.js\src\client\ClientManager.js:47:41)
    at Object.onceWrapper (events.js:315:30)
    at emitOne (events.js:116:13)
    at WebSocketConnection.emit (events.js:211:7)
    at WebSocketConnection.onClose (C:\Users\Bryan Hyunwoo Kim\Desktop\spambot-master\spambot\node_modules\discord.js\src\client\websocket\WebSocketConnection.js:390:10)
    at WebSocket.onClose (C:\Users\Bryan Hyunwoo Kim\Desktop\spambot-master\spambot\node_modules\ws\lib\event-target.js:124:16)
    at emitTwo (events.js:126:13)
    at WebSocket.emit (events.js:214:7)
    at _receiver.cleanup (C:\Users\Bryan Hyunwoo Kim\Desktop\spambot-master\spambot\node_modules\ws\lib\websocket.js:220:12)
    at Receiver.cleanup (C:\Users\Bryan Hyunwoo Kim\Desktop\spambot-master\spambot\node_modules\ws\lib\receiver.js:535:15)
(node:81152) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:81152) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

please help!

**Comment by ajmeese7** [comment]
I added the repository field in commit 584c641 and tested the code. It worked for me locally on Manjaro with Node v9.3.0.

Did you follow all the instructions in the README, including typing the correct login information into `config.js`? Let me know if either of you are still experiencing any problems @KimHWBryan @JaredStock20 

P.S. sorry it took me so long to respond. Usually GitHub sends me email notifications about everything, but I didn't get one about this issue until Jared commented on it

**Comment by JaredStock20** [comment]
I redownloaded the files and started over, exactly following your instructions, however, I still get this error:

npm WARN discord.js@11.3.2 requires a peer of bufferutil@^3.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of erlpack@discordapp/erlpack but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of node-opus@^0.2.7 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of opusscript@^0.0.6 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of sodium@^2.0.3 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of libsodium-wrappers@^0.7.3 but none is installed. You must install peer dependencies yourself.
npm WARN discord.js@11.3.2 requires a peer of uws@^9.14.0 but none is installed. You must install peer dependencies yourself.

audited 8 packages in 1.639s
found 0 vulnerabilities

I updated my config.json files and tried again, but I got the same error. Is there any other info or error codes I could get to make my problem clearer? If you want I could go through the steps again while recording my screen. I could be missing a step, however, I am pretty sure that I am not. All my programs are up to date too, so I don't think that is my problem.

**Comment by ajmeese7** [comment]
Oh no those errors aren't a problem. I got those warnings too but the program should still run as planned @JaredStock20 if you try running the start command and you experience any errors then let me know

**Comment by JaredStock20** [comment]
Oops, you're totally right! I did the start command, and it kinda worked. The bot is unresponsive when I do the /spam command. Another thing I noticed is that in the gif on the main page, the command prompt shows another message, whereas mine freezes after showing the "Ready to level up! message.

![thoiqandaadad](https://user-images.githubusercontent.com/34488610/42398077-42d8fb92-811d-11e8-86e7-8fad35eac968.PNG)

Sorry for my ignorance, I know HTML, so this is literally a foreign language for me! I really appreciate yr help
